src/test/compile-fail/issue-25386.rs

```diff
@@ -8,6 +8,10 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+// error-pattern: in this expansion of format_args!
+// error-pattern: in this expansion of print! (defined in <std macros>)
+// field `c_object` of struct `stuff::Item` is private
```

Comment:
I'm pretty sure I did this incorrectly. A couple errors in this testcase have errors that report on `<std macros>` and I'm not sure the best way to handle it.